torchmultimodal/models/flava/model.py

```diff
@@ -714,10 +716,11 @@ def __init__(
 
     def load_model(self) -> Any:  # type: ignore
         # TODO (T116682215): Network error due to FLAVA model relying on access to openAI
-        encoder = super().load_model(
-            "https://cdn.openai.com/dall-e/encoder.pkl", load_state_dict=False
+
+        encoder_state_dict = torch.hub.load_state_dict_from_url(
+            "https://cdn.openai.com/dall-e/encoder.pkl"
         )
-        self.encoder.load_state_dict(encoder.state_dict())  # type: ignore
+        self.encoder.load_state_dict(encoder_state_dict.state_dict())  # type: ignore
```

Comment:
wait why are we calling .state_dict() on something that's already a state_dict?

Reply:
its the same as before

Reply:
https://colab.research.google.com/drive/1adoRcJtQWq-zVQuVwyO-4m5Qfx6hlILx#scrollTo=0X0QoOwnV1Ei